services/ui-src/src/components/cards/EmailCard.test.tsx

```diff
@@ -13,34 +13,30 @@ const emailCardComponent = (
   />
 );
 
-describe("Test EmailCard", () => {
-  beforeEach(() => {
-    render(emailCardComponent);
-  });
+describe("<EmailCard />", () => {
+  describe("Test EmailCard", () => {
```

Comment:
do we need the nested describe if there are no other describe blocks?

Reply:
The second `describe` block here is to isolate `beforeEach`.